.travis.yml

```diff
@@ -8,6 +8,7 @@ install:
   - make install-travis
 
 script:
+  - OS_GOFLAGS="-p 1" travis_wait make build
```

Comment:
One wait gives us 20min, compare https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received. We will see if that's enough.